### DISCUSSION TITLE
[Bug]: scoop unable to install `time` on windows 11

### DISCUSSION THREAD

**Discussion by RayyanNafees** [ScoopInstaller/Main#5203] [Q&A]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

time

## Current Behavior

I tried to install the [`time` package](https://github.com/ScoopInstaller/Main/blob/master/bucket/time.json) via the [scoop installer](https://scoop.sh/#/apps?q=time)

But the installation failed in Windows 11 

When i tried to rerun `scoop install time` this error was shown:

![image](https://github.com/ScoopInstaller/Main/assets/59685062/0096e5b0-b35f-479e-98b4-8439832a8042)

## Expected behavior

To install correctly on win 11 and work as intended

### Steps to Reproduce

```console
PS> scoop install time
Installing 'time' (0.2018.07.25) [64bit] from main bucket
The SSL connection could not be established, see inner exception.
URL https://raw.githubusercontent.com/lukesampson/psutils/3290c04d7653ea8183a5e11d3d5a4e46716035d1/time.ps1 is not valid
```

### Possible Solution

I don't know am a beginner

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
0db9dfc7f (HEAD -> master, origin/master, origin/HEAD) strokesplus.net: Update to version 0.5.7.5

'java' bucket:
a524d2cd (HEAD -> master, origin/master, origin/HEAD) openjdk22: Update to version 22-20-ea

'main' bucket:
65045c185 (HEAD -> master, origin/master, origin/HEAD) updatecli: Update to version 0.64.1

'nerd-fonts' bucket:
99fe5dc (HEAD -> master, origin/master, origin/HEAD) SarasaGothic: Update to version 0.42.2
```

### Scoop Config

```console
last_update           scoop_repo                              scoop_branch
-----------           ----------                              ------------
20/10/2023 2:12:46 AM https://github.com/ScoopInstaller/Scoop master
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.8
PSEdition                      Core
GitCommitId                    7.3.8
OS                             Microsoft Windows 10.0.22631
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by rashil2000** [comment]
Are you able to access the link https://raw.githubusercontent.com/lukesampson/psutils/3290c04d7653ea8183a5e11d3d5a4e46716035d1/time.ps1 in a browser?
